woocommerce/woocommerce-android
# Display set tax button on created unpaid orders

Closes: #9854 


### Description

Context: pdfdoF-3GW-p2#comment-5031
The Set New Tax rate button was set to be displayed in Mode.Creation only and not on Edit mode. However, based on Cesar's comment, I have set it to be displayed on order creation plus on order editing whenever the order is not paid (does not have a paid date) 

### Testing instructions


1. Go to oders
2. open an order that has been paid
3. click on edit
4. make sure the `Set New Tax Rate` does not show
5. go to an order that has a pending payment
6. click on edit
7. make sure the button does show

### Images/gif

<img src = https://github.com/woocommerce/woocommerce-android/assets/30724184/0f3f9234-f579-4b34-8748-4f209a141e4d width = 350>
